app/src/main/java/tttTDD/Java/Board.java

```diff
@@ -33,8 +33,12 @@ public String[] getBoard() {
     }
 
     public int getFirstAvailableMove() {
-        String[] availableMoves = generateAvailableMoves(board);
-        return convertToLocation(availableMoves[0]);
+        Integer[] availableMoves = generateAvailableMoves(board);
+        return availableMoves[0];
+    }
+
+    public Integer[] getAvailableMoves() {
+        return generateAvailableMoves(board);
     }
```

Comment:
I agree that this is a superior method signature (both in name and (lack of) arguments), but why are you just wrapping the old one rather than replacing it?